config/http.conf

```diff
@@ -1,18 +1,33 @@
+daemon off;
+
+worker_processes auto;
+worker_shutdown_timeout 240s;
+
+events {
+  multi_accept        on;
+  worker_connections  16384;
+  use                 epoll;
+}
+
 http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
+  aio       threads;
+  aio_write on;
```

Comment:
Configures asynchronous file I/O which prevents the event loop from blocking.

See: https://www.nginx.com/blog/thread-pools-boost-performance-9x